distribution/docs/src/main/resources/content/_configuring/importing-exported-configurations-contents.adoc

```diff
@@ -6,20 +6,21 @@
 
 ==== Importing the Exported Configuration Settings
 
-[WARNING]
-====
-Importing of configurations must take place instead of performing installation for the first time and after performing OS-level hardening.
-====
-
 To import a previously exported configuration file from the ${command-console}:
 
+. Step through the installation process.
 . Launch the newly installed ${branding}.
-. Type in `migration:import <directory>`. This command reads the previously exported configuration file `exported-${project-version}.zip` from the specified directory. If no directory is specified it will default to `<${branding}_HOME>/exported`
+. Copy the previously exported file from your secure place to the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified with the `migration:import`. Create the directory if it does not exist.
```

Comment:
❓  Same question. Should we provide an example of the command with an alternative location?

Reply:
Same answer as above.